packages/components/badge/src/badge.vue

```diff
@@ -3,7 +3,7 @@
     <slot />
     <transition :name="`${ns.namespace.value}-zoom-in-center`">
       <sup
-        v-show="!hidden && (content || isDot)"
+        v-show="isShowBadge"
```

Comment:
```suggestion
        v-show="!hidden && (content || isDot)"
```
Please keep it, I will explain later.